sale_order_warehouse_location/models/sale_order.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2025 Manuel Regidor <manuel.regidor@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    @api.depends("partner_shipping_id")
+    def _compute_warehouse_id(self):
+        ret_vals = super()._compute_warehouse_id()
+        orders = self.filtered(
+            lambda o: o.partner_shipping_id
+            and (o.state in ["draft", "sent"] or not o.ids)
```

Comment:
Question, not an issue: why do we need to filter by `o.ids`?

Reply:
Hm, could you please give some examples? Just to elaborate.

Reply:
@ivs-cetmix So it seems not to be necessary. Changed